components/general/BadgeIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode } from 'react';
+import { StyleSheet, View, TouchableOpacity } from 'react-native';
+import { Badge } from 'react-native-elements';
+import { Spacing } from '../../theme';
+
+interface BadgeIconProps {
+  value: number;
+  children: ReactNode;
+  onPress: () => void;
+}
+const BadgeIcon: React.FC<BadgeIconProps> = ({ value, children, onPress }: BadgeIconProps) => {
+  return (
+    <TouchableOpacity style={styles.container} onPress={onPress}>
+      <View style={styles.row}>
+        {children}
+        {value > 0 && <Badge value={value} status={'error'} containerStyle={styles.badgeStyle} />}
```

Comment:
Think we should add some more styling to the badge. Remove white border and use red color defined in `Themes.tsx`. So that it is similar to badge used in the bottom bar😊